seeds/storage.yml

```diff
@@ -0,0 +1,67 @@
+--- !ruby/object:Library
```

Comment:
We don`t need to add this file to repository.

Reply:
fixed

Reply:
Please, remove it from repository

Reply:
removed

Reply:
done